tests/page-objects/common/about.wdio.page.js

```diff
@@ -0,0 +1,15 @@
+const userName = async () => await (await $('dt=User name')).nextElement();
+const partners = () => $('.partners');
+
+const getPartnerImage = async (name) => {
+  await (await partners()).waitForDisplayed({ timeout: 2000 });
```

Comment:
I think we should use the default (15000) timeout. If for some reason we think there should be two (short, default) then I think it'd be best to store it in the wdio.conf or constants file. Otherwise we'll get `{ timeout: 2000 }` `{ timeout: 5000 }` `{ timeout: 1000 }`  throughout the code base.

Reply:
Done.